nexusutils/dataconverter/helpers.py

```diff
@@ -169,6 +169,7 @@ def is_value_valid_element_of_enum(value, elem) -> Tuple[bool, list]:
     "NX_UINT": (np.ndarray, np.unsignedinteger),
     "NX_NUMBER": (int, float, np.ndarray, np.signedinteger, np.unsignedinteger, np.floating, dict),
     "NX_POSINT": (int, np.ndarray, np.signedinteger),  # > 0 is checked in is_valid_data_field()
+    "NX_COMPLEX": (complex, np.ndarray, np.complex128),
```

Comment:
Do you think we should add np.singlecomplex/np.csingle as well? And is this np.complex128 alias platform dependent? I see in the numpy docs that there is another alias which does not mention platform dependency. I guess this is just for x86_64. 

https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.csingle

Reply:
Yes it seems that `np.complex128` is platform depended. The platform independent equivalent would be `np.cdouble`, I think it's better to just use this instead of `np.complex128`.

Reply:
What about np.singlecomplex/np.csingle? Should I drop em in too? What do you think? h5py does support 8 byte complex numbers. So I'm hoping it supports np.csingle too.

Reply:
Ah yes, I think it's useful to add all valid types for h5py. I just had my use case in mind where I should only have `np.complex128`. I should also refactor the code parts from `np.complex128` to `np.cdouble`, so I'll also add the `np.csingle`.

Reply:
Thank you!